### PR TITLE
Added code changes to Kubernetes Services BRS to support Ready-to-run Script feature

### PR DESCRIPTION
**Pull request by saurabhgautam-msft**

### CODE REVIEW

File: Scripts/RemediationScripts/AutoRemediation.ps1 [hunk 1]
```diff
@@ -153,6 +153,17 @@ Enter the choice (1|2)";
...
                 }elseif ($control.ControlId -eq "Azure_AppService_DP_Use_Secure_TLS_Version") {
                     $commandString = $control.InitCommand + " -SubscriptionId " +  "`'" + $SubscriptionId +  "`'" +  " -Path " + "`'" + "FailedControls\" +  $SubscriptionId + ".json" + "`'" + " -PerformPreReqCheck"+ " -AutoRemediation" + " -TimeStamp " + "`'" + $timeStampString +  "`'";
+                }elseif ($control.ControlId -eq "Azure_KubernetesService_AuthN_Enabled_AAD") {
+                    Write-Host "[$($control.LoadCommand)] Bulk Remediation Script requires user inputs at some points to execute properly.`n" -ForegroundColor $([Constants]::MessageType.Warning)
+                    $proceedWithRemediation = Read-Host -Prompt "Do you want to proceed with remediation for the control [$($control.ControlId)]? (Y|N)"
+                    Write-Host $([Constants]::SingleDashLine)
+                    if(($proceedWithRemediation -ne 'Y') -and ($proceedWithRemediation -ne 'y'))
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/AutoRemediation.ps1:160]: PowerShell is case-insensitive, so case check might not be needed

File: Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1 [hunk 2]
```diff
@@ -146,86 +159,164 @@ function Enable-AADForKubernetes
...
-            break
+            Write-Host "Error occurred while setting up prerequisites. Error: [$($_)]" -ForegroundColor $([Constants]::MessageType.Error)
+            Write-Host $([Constants]::DoubleDashLine)
+            return
         }
     }
-
-    # Get current Azure account context.
+    else
+    {
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1:198]: If user run with $PerformPreReqCheck, then we will not show Validate the user step.

Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1:198]: We will, it just that the heading of step is changing.

Comment (TarunKrShukla) [reply Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1:198]: What will be the heading in that case?

Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1:198]: Validate the user

Comment (TarunKrShukla) [reply Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1:198]: ok

**Review by TarunKrShukla — APPROVED** [review]
Looks good.

**Pull request merged**